### DISCUSSION TITLE
gd-77 Power setting wit word doesn't work

### DISCUSSION THREAD

**Discussion by radeknoone** [hmatuschek/qdmr#210] [General]
I'm using gd-77 with opengd77 software and if I use following channel configuration, then the configuration in the transmiiter has the power: 

PMR 1: 50mW
PMR 2: 50mW
PMR 3: 50mW
PMR 4: 5W
PMR 5: 5W

I would expect that the Power leveles Min, Low, Mid, High, Max should be different. Is there any possiblity to define the power by number in the Watts?

```
Analog  Name                Receive   Transmit   Power Scan TOT RO Admit  Squelch RxTone TxTone Width APRS
1       "PMR 1"             446.00625 446.00625  Min   -    -   -  -      1       -      -      12.5  -
2       "PMR 2"             446.01875 446.01875  Low   -    -   -  -      1       -      -      12.5  -
3       "PMR 3"             446.03125 446.03125  Mid   -    -   -  -      1       -      -      12.5  -
4       "PMR 4"             446.04375 446.04375  High  -    -   -  -      1       -      -      12.5  -
5       "PMR 5"             446.05625 446.05625  Max   -    -   -  -      1       -      -      12.5  -

**Comment by hmatuschek** [comment]
Actually, you can set the power levels for the OpenGD77 firmware in more detail. However, it is not possible in the old table-based format. Since the version 0.9.0 there is a new extensible codeplug format (https://dm3mat.darc.de/qdmr/manual/ch03.html) based on YAML. This allowed me to implement device-specific extensions. There is one extension for the OpenGD77 firmware to set the power-levels (https://dm3mat.darc.de/qdmr/manual/ch04.html#idp27). Please note that the extension will override the generic (min, low, mid, high, max) settings.

**Comment by radeknoone** [comment]
Hi Hannes,

thank you for quick and explanatory response and also for great piece of software! This ticket can be closed. Unfortunately, I'm not able to use higher version than 8, as Im using Ubuntu or Snap packages. When I tried to compile version 9 or 10, there was some errors.

R.
